code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm

```diff
@@ -52,7 +52,7 @@
 	GLOB.round_statistics.runner_savage_attacks++
 	SSblackbox.record_feedback("tally", "round_statistics", 1, "runner_savage_attacks")
 	M.attack_alien_harm(src, extra_dam, FALSE, TRUE, TRUE, TRUE) //Inflict a free attack on pounce that deals +1 extra damage per 4 plasma stored, up to 35 or twice the max damage of an Ancient Runner attack.
-	use_plasma(extra_dam * 5) //Expend plasma equal to 4 times the extra damage.
+	use_plasma(extra_dam * 4) //Expend plasma equal to 4 times the extra damage.
```

Comment:
I like how you didn't have to update the comment here because whoever nerfed it last failed to.

Reply:
I only changed it because of the comment implying it's 4, lol.